internal/controller/istio/controller.go

```diff
@@ -125,6 +121,8 @@ func (c *Controller) connectIstio() {
 			sm, err := c.newSecretManager()
 			if err != nil {
 				controllerLog.Errorf("failed to create SecretManager %s %v", c.options.IstiodAddr, err)
+				time.Sleep(5 * time.Second)
+				continue
```

Comment:
Could you please fix the go lint issue?

superfluous-else: if block ends with a continue statement, so drop this else and outdent its block (revive)

Reply:
ok has fixed